### ISSUE TITLE
Accessible user profile and co-creations

### ISSUE THREAD

**Issue by AltiMario** [e-Learning-DAO/POCRE#50]
We want to give external (not logged) users "access" to view registered users info.
![image](https://user-images.githubusercontent.com/598726/193416316-136f299a-0549-4a07-8016-87d34650cbdb.png)

The user profile (linked to the wallet) has to be publicly accessible for everybody, together with the co-creations he/she did.
![image](https://user-images.githubusercontent.com/598726/193416364-e707b40f-899c-49d6-bdeb-6d3738fee46f.png)
 
To recap, the overall idea is to have some sort of "open linkedin" where everybody can see the Alice operas.
The Alice's open information will be:
-bio with wallet reputation 
-list and details of creations
-list and details of materials recognised as author

**Comment by Mahdi-Soultana** [comment]
totally agree and it's great idea to encourage users to sign up .
